arm-cdn/2016-10-02/swagger/cdn.json

```diff
@@ -343,6 +343,47 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cdn/profiles/{profileName}/getSupportedOptimizationTypes": {
```

Comment:
Does this return a specific subset of the set in  the `enum` in  `OptimizationType`  ? (ie, it's not just the whole list every time, right?)